src/shogun/machine/IterativeLinearMachine.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Shubham Shukla
+ */
+
+#ifndef _ITERATIVELINEARMACHINE_H__
+#define _ITERATIVELINEARMACHINE_H__
+
+#include <shogun/lib/config.h>
+
+#include <shogun/lib/SGVector.h>
+#include <shogun/lib/common.h>
+#include <shogun/machine/LinearMachine.h>
+
+namespace shogun
+{
+	class CFeatures;
+	class CLabels;
+
+	class CIterativeLinearMachine : public CLinearMachine
```

Comment:
some docs describing that this class implements a model whose training can be stopped, and in particular resumed, at any time